### ISSUE TITLE
[PM-1736] Pegasus pollutes the job env when getenv=true

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#1850]
When a user set getenv=true, they get not only their own environment but also a bunch stuff from Pegasus. This is particular bad with PYTHONPATH as it is set based on whatever Python3 we found on the submit host. 

This issue is still being explored, but it seems like saving the original PYTHONPATH in pegasus-python-wrapper and resetting back to it in pegasus-dagman would solve the problem.

**Reporter:** @rynge
**Resolution:** Fixed
**Watchers:**
@rynge
@vahi

**Comment by mayani** [comment]
**Author:** @rynge

I have committed a fix which resets PYTHONPATH and CLASSPATH to what they were before Pegasus was invoked. The reset happens inside pegasus-dagman, when condor_dagman is invoked. Variables named PEGASUS_ORIG_{NAME} will be used to set {NAME} back the orignal value, and after that, any PEGASUS_* variable is cleared out. I also added a test case for this called 048-ligo-getenv, which is now green.
